tokenid_test.go

```diff
@@ -22,7 +22,7 @@ func TestDeriveTokenId(t *testing.T) {
 		{
 			name:     "Invalid ENS domain",
 			expected: "",
-			input:    "foo.bar",
+			input:    "invalid.domain",
```

Comment:
I had to change this because for some reason, even though it isn't a registered domain, the contract returns the `OffchainLookup` revert due to the `bar` TLD's external resolver.